src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
When you write new ..() -- memory is allocated.
Each method invocation you will be creating a new object, that performs same operations.
This object should be final field.